pkg/utils/yaml_utils.go

```diff
@@ -8,6 +8,14 @@ import (
 	"github.com/cloudposse/atmos/pkg/schema"
 )
 
+var (
+	AtmosYamlTags = []string{
+		"!terraform.output",
+		"!exec",
+		"!template",
```

Comment:
Use the consts from the config package so if we add/refactor we don't have to do it in multiple places.